apps/nextra/pages/en/build/indexer/indexer-sdk/quickstart.mdx

```diff
@@ -32,24 +32,27 @@ You also need the following tools:
 
 - Rust 1.79: [Installation Guide](https://www.rust-lang.org/tools/install)
 - Cargo: [Installation Guide](https://doc.rust-lang.org/cargo/getting-started/installation.html#install-rust-and-cargo)
-- Diesel CLI: [Installation Guide](https://diesel.rs/guides/getting-started.html)
 
-We use PostgreSQL as our database in this tutorial. You’re free to use whatever you want, but this tutorial is geared
+We use PostgreSQL as our database and Diesel as our ORM in this tutorial. You’re free to use whatever you want, but this tutorial is geared
 towards PostgreSQL for the sake of simplicity. We use the following database configuration and tools:
 
+### PostgreSQL Installation
 [PostgreSQL](https://www.postgresql.org/download/)
+1. `brew install libpq` ([this is a postgres C API library](https://formulae.brew.sh/formula/libpq)). Also perform all export commands post-installation
```

Comment:
The reason I say this is I got hung up because on Linux, it's `libdev-pq`

But, let's just put a `Mac` section here, and then we can come back and put similar for at least one Linux.